x/derivatives/keeper/perpetual_futures.go

```diff
@@ -253,7 +252,7 @@ func (k Keeper) ReportLiquidationNeededPerpetualFuturesPosition(ctx sdk.Context,
 		return nil
 	}
 
-	return errors.New("no liquidation needed")
```

Comment:
If it is not subject to ReportLiquidation, it should end the process, not an error